pkg/controller/constraint/constraint_controller.go

```diff
@@ -364,6 +364,20 @@ func (r *ReconcileConstraint) Reconcile(ctx context.Context, request reconcile.R
 				return reconcile.Result{}, err
 			}
 		}
+		isAPIEnabled, groupVersion := transform.IsVapAPIEnabled(&log)
+		if isAPIEnabled {
+			currentVapBinding, err := vapBindingForVersion(*groupVersion)
+			if err != nil {
+				return reconcile.Result{}, err
+			}
+			vapBindingName := fmt.Sprintf("gatekeeper-%s", instance.GetName())
```

Comment:
nit: can we move generating the binding name from the constraint into its own function?

I think we do this string format more than once (to handle create/update), so should help with uniformity.